cmd/devtools/config/networks.go

```diff
@@ -0,0 +1,305 @@
+	if err := viper.Unmarshal(&config); err != nil {
+		log.Fatalf("Unable to decode into struct, %v", err)
+		panic(err)
+	}
+
+	return config
+}
+
+// CreateNetworksConfig creates a networks configuration file at
+// the given path, populating the file with the network defaults, and overriding
+// the default local denom and local sequencer network chain id .
+// It will skip initialization if the file already exists. It will panic if the
+// file cannot be created or written to.
+func CreateNetworksConfig(path, localSequencerChainId, localDefaultDenom string) {
+
+	_, err := os.Stat(path)
+	if err == nil {
+		log.Infof("%s already exists. Skipping initialization.\n", path)
+		return
+	}
+	// Create an instance of the Config struct with some data
+	config := DefaultNetworksConfigs()
+	config.Local.DefaultDenom = localDefaultDenom
+	config.Local.SequencerChainId = localSequencerChainId
+
+	// Open a file for writing
+	file, err := os.Create(path)
+	if err != nil {
+		panic(err)
+	}
+	defer file.Close()
+
+	// Encode the struct to TOML and write to the file
+	if err := toml.NewEncoder(file).Encode(config); err != nil {
+		panic(err)
+	}
+	log.Infof("New network config file created successfully: %s\n", path)
+}
+
+// CreateBaseConfig creates a networks configuration file at
+// the given path, populating the file with the network defaults, and overriding
+// the default local denom and local sequencer network chain id .
+// It will skip initialization if the file already exists. It will panic if the
+// file cannot be created or written to.
+func CreateBaseConfig(path, instance string) {
+
+	_, err := os.Stat(path)
+	if err == nil {
+		log.Infof("%s already exists. Skipping initialization.\n", path)
+		return
+	}
+	// Create an instance of the Config struct with some data
+	config := NewBaseConfig(instance)
+
+	// Open a file for writing
+	file, err := os.Create(path)
+	if err != nil {
+		panic(err)
+	}
+	defer file.Close()
+
+	// Encode the struct to TOML and write to the file
+	if err := toml.NewEncoder(file).Encode(config); err != nil {
+		panic(err)
+	}
+	log.Infof("New network config file created successfully: %s\n", path)
+}
+
+// LoadBaseConfigOrPanic loads the BaseConfig from the given path. If the file
+// cannot be loaded or parsed, the function will panic.
+func LoadBaseConfigOrPanic(path string) BaseConfig {
+	viper.SetConfigFile(path)
+
+	if err := viper.ReadInConfig(); err != nil {
+		log.Fatalf("Error reading config file, %s", err)
+		panic(err)
+	}
+
+	var config BaseConfig
+	if err := viper.Unmarshal(&config); err != nil {
+		log.Fatalf("Unable to decode into struct, %v", err)
+		panic(err)
+	}
+
+	return config
+}
+
+// GetEnvOverrides returns a slice of environment variables that can be used to
+// override the default environment variables for the network configuration. It
+// uses the BaseConfig to properly update the ASTRIA_COMPOSER_ROLLUPS env var.
+func (n NetworkConfig) GetEnvOverrides(bc BaseConfig) []string {
```

Comment:
I've got a recommendation for a bit of a refactor involving this and the service log level overrides. 

![image](https://github.com/astriaorg/astria-cli-go/assets/5599073/5833133a-6e09-48ea-8b0e-a7998e8e6ee4)

Reply:
heh, that image was automatically copied from OneNote. Kinda neat. Here's the code for reference
```go
serviceLogLevelOverrides := config.GetServiceLogLevelOverrides()
endpointOverrides := networkConfigs.Local.GetEnvOverrides(baseConfig)
config := config.MergeConfigs(baseConfig, endpointOverrides, serviceLogLevelOverrides)
```

Reply:
This requires some refactoring related to my comment on this line about `MergeConfig`
https://github.com/astriaorg/astria-cli-go/pull/79/files#diff-f8fdb678586e222c489567b41972701140b4b6031df1c07e24ce6217a29b0fc8R275

Reply:
I think my wording of the first bullet point was ambiguous. I mean move the logic for serviceLogLevelOverrides to its own function similar to how you have `GetEnvOverrides`, not to move the logic for serviceLogLevelOverrides to GetEnvOverrides literally.
Probably not an issue, just wanted to make sure I wasn't confusing.

Reply:
Got these changes in. I think things are easier to understand now!